drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -524,312 +500,327 @@
-      <t>This section defines a method that uses a symmetric key based signature
-      algorithm for integrity protection of IOAM-Data-Fields.
-      In case of performance
-      concerns, the method can be applied to a subset of the traffic by using
-      sampling of data.</t>
-
-      <t>The symmetric key based signature algorithm MUST use SHA-256
-      (<xref target="SHS"/>) as the Digest Algorithm, and MUST use Advanced
-      Encryption Standard (<xref target="AES"/>) with a key length of 256 bits
-      and the Galois/Counter Mode (<xref target="NIST.800-38D"/>) as the
-      Signature Algorithm (AES-256-GCM). The corresponding Signature Suite
-      Identifier is 1, as defined in <xref target="IOAM-IP-registry"/>. As a
-      consequence, the signature consumes 32 octets. The Integrity Protection
-      Method is defined by the following steps:
-
-      <list style="numbers">
-        <t>The encapsulating node creates a nonce and stores it in the Nonce
-        field of the IOAM Integrity Protection Header (the Nonce length
-        field is set accordingly). The Signature-suite field is set to 1.
-        The signature is generated over the hash of header fields (see
-        <xref target="TraceOptTypeHdrFields"/>,
-        <xref target="POTOptTypeHdrFields"/>,
-        <xref target="E2EOptTypeHdrFields"/>, or
-        <xref target="DEXOptTypeHdrFields"/>, for the exact list of header
-        fields to include in the signature, depending on the IOAM Integrity
-        Protected Option-Type) and IOAM-Data-Fields it has inserted, i.e.,
-        sign(hash(Header-Fields || IOAM-Data-Fields)), with the Nonce field
-        provided as the nonce. IOAM-Data-Fields supposed to be modified by
-        other IOAM nodes on the path MUST be excluded from the signature
-        (e.g., the POT Cumulative field). The signature is stored in the
-        Signature field of the IOAM Integrity Protection Header.</t>
-
-        <t>A transit node generates a signature over the hash of
-        IOAM-Data-Fields it has inserted, i.e., sign(hash(IOAM-Data-Fields)),
-        with the Signature field provided as the nonce. IOAM-Data-Fields
-        modified in-place by the transit node MUST be excluded from the
-        signature (e.g., the POT Cumulative field). The signature is stored
-        in the Signature field of the IOAM Integrity Protection Header.
-
-        <list style="bullets">
-          <t>If the transit node does not insert IOAM-Data-Fields
-          (e.g., it only modifies IOAM-Data-Fields in-place, or does nothing),
-          then the transit node MUST NOT generate a signature and MUST NOT
-          update the Signature field.</t>
-        </list>
+      <t>This document defines a new method that uses a symmetric key based
+      algorithm for the integrity protection of IOAM-Data-Fields. In case of
+      performance concerns, the method can be applied to a subset of the traffic
+      by using sampling of data.</t>
```

Comment:
I'd suggest to remove the second sentence (i.e., "In case of performance concerns, ..."), since we already discuss it in the Security section. But still, your remark makes sense and I'll add a sentence in the Security section. Sounds good?